exist-core/src/test/java/org/exist/xquery/functions/fn/transform/FunTransformTest.java

```diff
@@ -166,7 +166,11 @@ private static class SourceWithLocation implements Source {
             this.location = location;
         }
 
-        // implementation solely to conform to interface for test use
```

Comment:
I guess when moving this comment into the method body all should be good..

Reply:
I turned it into a method document comment like the other(s)